### PR TITLE
Post merge hook

### PR DESCRIPTION
**Pull request by alebian**
## Summary

Added a script for post-merge git hook. This will execute the regular commands you need after a git pull

### CODE REVIEW

File: post-merge [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
```
Comment (epintos) [inline post-merge]: Why not adding it to the scripts folder?

**Comment by epintos** [comment]
@alebian Any update on this?

**Comment by epintos** [comment]
@alebian ping

**Comment by mishuagopian** [comment]
@alebian @hdf1986 are we merging this or should we close it?

**Comment by alebian** [comment]
@epintos done!!1

**Pull request merged**